src/components/individualArticlePage/CommentsList.jsx

```diff
@@ -1,24 +1,49 @@
 import Card from 'react-bootstrap/Card';
 import ListGroup from 'react-bootstrap/ListGroup';
 import Badge from 'react-bootstrap/Badge';
+import { CurrentUserContext } from '../../contexts/CurrentUser';
+import { useContext} from 'react';
+import deleteComment from '../../utils/deleteComment';
+
+function Comments({comments, commentDeletedId, setCommentDeletedId}) {
+    const {currentUser} = useContext(CurrentUserContext)
+
+    const handleClick = (comment_id) => {
+        setCommentDeletedId(comment_id)
+        deleteComment(comment_id)
+        .then((response) => {
+            if(response.status === 204){
+                setCommentDeletedId(comment_id)
```

Comment:
is commentDeletedId not already comment_id at this stage?

Reply:
Currently we're making the delete request, but we don't see the effects permanently until we refresh. (If I click delete on two comments, the first one comes back). How can we update the list of comments upon deletion?